airflow/www/views.py

```diff
@@ -740,13 +742,12 @@ def index(self):
                 current_dags = all_dags
                 num_of_all_dags = all_dags_count
 
-            dags = (
-                current_dags.order_by(DagModel.dag_id)
-                .options(joinedload(DagModel.tags))
-                .offset(start)
-                .limit(dags_per_page)
-                .all()
+            sort_column = DagModel.__table__.c[arg_sorting_key]
```

Comment:
This should add a fallback if the sorting key is invalid.

Reply:
Done, if sort_column is not found (None), we do not apply the order_by clause.